### PR TITLE
Ab#69372 adding azure ad

### PR DESCRIPTION
**Pull request by tropical-turtle**
Added Azure Active directory integration.

### CODE REVIEW

**Review by AlexandreSanscartier — CHANGES_REQUESTED** [review]
A few changes nothing dramatic good work!

**Review by AlexandreSanscartier — CHANGES_REQUESTED** [review]
Looks good just take a look at my previous comments and let me know what you think before we close this.

File: src/CSF.SRDashboard.Client/Services/GraphApiService.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+using Microsoft.Extensions.Configuration;
+using Microsoft.Identity.Web;
+using System;
+using System.Linq;
+using System.Net.Http;
+
+
+namespace CSF.SRDashboard.Client.Services
+{
+    public class GraphApiService : IGraphApiService
```
[the remark this answers is not in the document]
Comment (tropical-turtle) [reply src/CSF.SRDashboard.Client/Services/GraphApiService.cs:11]: Good point.

File: src/CSF.SRDashboard.Client/Shared/LoginDisplay.razor.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        private string photoDataString;
+
+        [Inject]
+        protected IGraphApiService graphService { get; set; }
+
+        protected override void OnInitialized()
+        {
+            userDisplayName = graphService.GetUserDisplayName();
+            photoDataString = graphService.GetUserPhotoData();
+        }
```
[the remark this answers is not in the document]
Comment (tropical-turtle) [reply src/CSF.SRDashboard.Client/Shared/LoginDisplay.razor.cs:22]: Good point.

File: src/CSF.SRDashboard.Client/Services/IGraphApiService.cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+﻿
+namespace CSF.SRDashboard.Client.Services
+{
+    public interface IGraphApiService
```
[the remark this answers is not in the document]
Comment (tropical-turtle) [reply src/CSF.SRDashboard.Client/Services/IGraphApiService.cs:4]: Good point.

**Review by AlexandreSanscartier — APPROVED** [review]
Good work!

**Pull request merged**